content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -345,7 +345,7 @@ either be a snapshot file from a previous backup operation, or from a remaining
 When restoring the cluster, use the `--data-dir` option to specify to which folder the cluster should be restored:
 
 ```shell
-etcdutl --data-dir <data-dir-location> snapshot restore snapshot.db
+etcdctl --data-dir <data-dir-location> snapshot restore snapshot.db
```

Comment:
I think the existing content is correct.

Reference: 
- https://etcd.io/docs/v3.6/op-guide/recovery/#restoring-a-cluster
- https://github.com/etcd-io/etcd/blob/main/etcdutl/README.md

Reply:
Thanks for clarifying, I will close the PR.